packages/sdk/src/dataProtector/IExecDataProtector.ts

```diff
@@ -242,6 +245,25 @@ class IExecDataProtector {
   rentProtectedData = (
     args: RentProtectedDataParams
   ): Promise<SuccessWithTransactionHash> => rentProtectedData(args);
+
+  setProtectedDataForSale = (args: {
+    protectedDataAddress: AddressOrENS;
+    priceInNRLC: number;
+  }): Promise<SuccessWithTransactionHash> => {
```

Comment:
Note that we don't require a `collectionTokenId` param since we can get it from the protected data.

Reply:
yeah good, idea we should verify that we can do the same thing in other functions that was already written